packages/hash/api/src/model/util.ts

```diff
@@ -51,4 +59,150 @@ export const RESTRICTED_SHORTNAMES = [
   "v2",
 ];
 
-export const NIL_UUID = "00000000-0000-0000-0000-000000000000";
+export const nilUuid = "00000000-0000-0000-0000-000000000000" as const;
+
+/**
+ * @todo: create workspace types in an account that's dedciated to
+ * the HASH workspace. For now we're just chucking them in the root account.
+ */
+export const workspaceAccountId = nilUuid;
+
+export const worskspaceTypesNamespaceUri =
+  "https://example.com/@workspace/types";
```

Comment:
Yeah, this will definitely not be the longterm way of determining the namespace URI. I forgot we had `SYSTEM_ACCOUNT_SHORTNAME`, should I rename this and related env vars to `WORKSPACE_ACCOUNT_SHORTNAME`? I think using "system" makes it ambiguous whether we're referring to the graph root account, or the workspace account.

Reply:
Sounds like a good idea, yeah. We're using that ENV var in a couple places, so perhaps a search and replace will suffice (this breaks some Terraform scripts in another repo as well)

Reply:
Cool, I've renamed them using find and replace across the repo, and made use of `WORKSPACE_ACCOUNT_SHORTNAME` in https://github.com/hashintel/hash/pull/919/commits/085bc6b4cd42caed3856be654586f77a23036cd1